synapse/python_dependencies.py

```diff
@@ -87,6 +87,8 @@
     # with the latest security patches.
     "cryptography>=3.4.7",
     "ijson>=3.1",
+    # Used for encrypted push
+    "pycryptodome>=3",
```

Comment:
Is this used now? I can't see it in the PR changes with a naive search

Reply:
It is currently used for the aes implementation. Can look into it tomorrow if cryptography has one, too (not sure why soru didn't do that before? some of this code is over 9 months old by now, after all ^^)

Reply:
Seems like it would be worthwhile to check — thanks.

Reply:
yup, cryptography does aes, too, while their API is a bit more cumbersome. Did update the code to use that, though!

Also, soru hid the entire thing behind a feature flag now, too.